sql/core/src/main/java/org/apache/spark/sql/execution/vectorized/ColumnVector.java

```diff
@@ -387,6 +449,11 @@ private void throwUnsupportedException(int requiredCapacity, Throwable cause) {
   public abstract byte getByte(int rowId);
 
   /**
+   * Gets values from [rowId, rowId + count) to [dst, dst + count)
```

Comment:
or shall we make the api `public abstract byte[] getBytes(int rowId, int count);`?